src/lib/containers/works_for_you.tsx

```diff
@@ -203,33 +224,75 @@ const styles = StyleSheet.create<Styles>({
 export default Relay.createContainer(WorksForYou, {
   initialVariables: {
     totalSize: PageSize,
+    selectedArtist: null,
+    showSpecialNotification: false,
   },
   fragments: {
-    me: () => Relay.QL`
-      fragment on Me {
-        notifications_connection(first: $totalSize) {
-          pageInfo {
-            hasNextPage
-          }
-          edges {
-            node {
-              ${Notification.getFragment("notification")}
+    viewer: () => Relay.QL`
+      fragment on Viewer {
+        me {
+          notifications: notifications_connection(first: $totalSize) {
+            pageInfo {
+              hasNextPage
+            }
+            edges {
+              node {
+                ${(Notification.getFragment("notification"))}
+              }
             }
           }
         }
+
+        selectedArtist: artist(id: $selectedArtist)
+                          @include(if: $showSpecialNotification) {
+          name
+          image {
+            resized(height: 80, width: 80) {
```

Comment:
Because of 🐔 and 🥚 problems, I moved the resize/crop URL logic from MP into Emission in our image view component. What it does is wait to be laid out so that it knows its actual size, multiply by screen pixel ratio, and then fetch the actual resized/cropped image.